### ISSUE TITLE
Undersøg forskelle på nuværende implementering og papir

### ISSUE THREAD

**Issue by asgerius** [asgerius/rl-rubiks#56]
# Umiddelbare forskelle:
- Batchnorm?
- Softmax på policy i MCTS?
- Initialisering
- Inkludering af løste tilstande
